<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Ekran" name="wEkran" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>49</y>
      <width>801</width>
      <height>521</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="LineButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CircButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Circular</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Curve">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>New Curve</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Add Point</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Delete Point</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Move Point</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="FillButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Color">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="Polygon">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>New Polygon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Add point</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Delete point</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Move point</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ekran</class>
   <extends>QWidget</extends>
   <header location="global">ekran.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
